appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -471,17 +471,28 @@ class LoggedInFlowNode @AssistedInject constructor(
         waitForNavTargetAttached { navTarget ->
             navTarget is NavTarget.RoomList
         }
-        attachChild<RoomFlowNode> {
-            backstack.push(
-                NavTarget.Room(
-                    roomIdOrAlias = roomIdOrAlias,
-                    serverNames = serverNames,
-                    trigger = trigger,
-                    initialElement = RoomNavigationTarget.Messages(
-                        focusedEventId = eventId
+
+        val isRoomAlreadyDisplayed = backstack.elements.value.any {
```

Comment:
I think we should only check the active element, not if the room is in the backstack?

Reply:
Yes, you're right. I was using this because I thought the backstack was shared with the one in `RoomFlowNode`, but with the current implementation we only need the active element check.